README.md

```diff
@@ -0,0 +1,47 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# OOP School Library
+
+> A ruby on rails app for maintaining personnel budgets.
+
+## Live Link
+[LIVE DEMO](https://budgeting-app-rails.herokuapp.com/)
+## Built With
+
```

Comment:
- [ ] Kindly, make sure that you customize your project title with a right title.